presto-main/src/test/java/io/prestosql/operator/TestAggregationOperator.java

```diff
@@ -91,7 +91,7 @@ public void tearDown()
     public void testMaskWithDirtyNulls()
     {
         // Ensures that the operator properly tests for nulls in the mask channel before reading its value
-        InternalAggregationFunction countBooleanColumn = metadata.getAggregateFunctionImplementation(
```

Comment:
@martint was this intentionally like this?
otherwise I guess the `getAggregateFunctionImplementation` call be removed.

Reply:
Thanks, remove call already.